### DISCUSSION TITLE
rioxarray equivalence of gdal.buildVRT

### DISCUSSION THREAD

**Discussion by huayue21** [corteva/rioxarray#626] [General]
what is the rioxarray way to implement  same function as gdal.buildVRT.
Is there any function available so I dont have to import rasterio.gdal?

Thanks,

**Comment by snowman2** [comment]
See: https://github.com/rasterio/rasterio/issues/2573

**Reply by huayue21** [reply]
Got it. Thanks. This community is extremely friendly!

**Comment by huayue21** [comment]
Hi @snowman2 
I used merge_arrays from rioxarray.merge to replace the gdal.buildVRT. 
need to do open_rasterio() seqentially to feed the function merge_arrays. 
But I avoid saving the xml file in  gdal.buildVRT.
